### DISCUSSION TITLE
E-mail after end-of-development

### DISCUSSION THREAD

**Discussion by MikeMcC399** [corona-warn-app/cwa-website#3505] [Q&A] [answered]
The FAQs say that the e-mail address will be disabled.

- [CODE_OF_CONDUCT.md](https://github.com/corona-warn-app/cwa-website/blob/master/CODE_OF_CONDUCT.md)
- [CONTRIBUTING.md](https://github.com/corona-warn-app/cwa-website/blob/master/CONTRIBUTING.md)
- [SECURITY.md](https://github.com/corona-warn-app/cwa-website/blob/master/SECURITY.md)

each link to the e-mail address corona-warn-app.opensource@sap.com. Presumably this is the e-mail address which https://www.coronawarn.app/en/faq/results/#ramp_down_mailbox implies will be disabled on May 31, 2023.

There should be an active e-mail address maintained for Code of Conduct and Security issues if this repository remains active (i.e. unarchived) past May 31, 2023.

**Comment by Ein-Tim** [comment]
cc @GisoSchroederSAP

**Comment by MikeMcC399** [comment]
Or if no e-mail address is maintained, then the documents would need to be reviewed and changed.

**Comment by Ein-Tim** [comment]
I will raise this issue internally.

**Comment by Ein-Tim** [comment]
Thank you for raising this, Mike! You will see more PRs in the next days addressing this and other changes that need to be done as SAP ends its work on the CWA website.

**Reply by MikeMcC399** [reply, marked as answer]
@Ein-Tim 

- Your PR https://github.com/corona-warn-app/cwa-website/pull/3513 answers this issue.

Thanks!
